### DISCUSSION TITLE
Copy influxdb to other (new) instance

### DISCUSSION THREAD

**Discussion by bennysweb** [solectrus/solectrus#2297] [Q&A]
Hi,
how can I copy the data from influxdb to a new influxdb instance? Background of my question is that I want to migrate my solectrus installation from a raspberry to a docker container on my synology nas. New clean installation of solectrus in docker works fine, import of the .csv files from senec as well. But the temperatures and all the other data from the last year wich are not provided in the .csv files are logically missing and I don't want to lose them. 
Best regards,
Benny

**Comment by ledermann** [comment]
Maybe it could work like this:

1. Shut down the entire SOLECTRUS stack (all containers) on the Raspberry Pi. The databases must not be running for the next steps.
2. Ensure there is not already a SOLECTRUS instance on Synology
3. Copy the `solectrus` folder with all subfolders from Raspberry Pi to Synology. This includes `.env`, `docker-compose.yml` and the database subfolders `postgresql/*`, `/redis/*` and `influxdb/*`. Ensure to not overwrite existing folders on the Synology, this would cause big trouble (hence step 2)
4. Start up the stack on the Synology by `docker-compose up -d` and look closely at the logs of the database containers.

I'm not fully sure that this works because the architectures are different (Raspberry Pi is `aarch64`, Synology is `x86_64`). In case of problems, you can try to back up and restore the databases in the official way.

**Reply by bennysweb** [reply]
Export the database and import on the new system works perfect für me. Thanks for the tip. 
Benny
